recipes/graphics/fonts/dejavu.yaml

```diff
@@ -0,0 +1,27 @@
+inherit: [install]
+
+metaEnvironment:
+    PKG_VERSION: "2.37"
+
+checkoutSCM:
+    scm: url
+    url: ${SOURCEFORGE_MIRROR}/dejavu/${PKG_VERSION}/dejavu-fonts-ttf-${PKG_VERSION}.tar.bz2
+    digestSHA1: 7fa15e7b9676fc3915338c41e76ad454c344fff5
+    stripComponents: 1
+
+buildScript: |
+    FONTS="DejaVuSansMono*.ttf DejaVuSans.ttf DejaVuSans-*.ttf DejaVuSerif.ttf DejaVuSerif-*.ttf DejaVuSansCondensed*.ttf DejaVuSerifCondensed*.ttf"
+    FONTCFG="20-unhint-small-dejavu-sans-mono.conf 57-dejavu-sans-mono.conf 20-unhint-small-dejavu-sans.conf 57-dejavu-sans.conf 20-unhint-small-dejavu-serif.conf 57-dejavu-serif.conf"
```

Comment:
Not enforced yet but could we break such long lines? Also, using arrays (`FONTS=( "DejaVuSansMono*.ttf" ...)`)  would be more bash-like.

Reply:
done

Reply:
Thanks. But the patterns are now unquoted. Thus, the shell will try apply *pathname expansion*. I guess the patterns won't match but we should play safe here. Also the `${FONTS[@]}` should have quotes: -> `"${FONTS[@]}"` (FONTCFG likewise).

Reply:
done